### DISCUSSION TITLE
Automatic Light/Dark theme based on OS/browser setting

### DISCUSSION THREAD

**Discussion by andreynering** [grafana/grafana#55568] [General]
Hi there,

In the settings we have the Default, Light and Dark settings:

<img width="239" alt="Screen Shot 2022-09-21 at 09 56 04" src="https://user-images.githubusercontent.com/7011819/191509869-4e9479ef-6df0-48ce-b6d6-2c5e59c21cc4.png">

I never really understood what the Default setting is supposed to mean, because it's Dark for me.

But what I really miss is a setting that uses the OS/browser setting, i.e., adjusts automatically based on the global setting.

Several web apps support that, GitHub being one example:

<img width="804" alt="Screen Shot 2022-09-21 at 10 00 08" src="https://user-images.githubusercontent.com/7011819/191510549-213a8ef9-8f25-4465-bb83-5a7bbf5124da.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
